raster-foundry/raster-foundry
# Add Project Layer Export Create UI

## Overview

This PR adds the project layer export creation UI using the new AOI creation component that defines an area for export.

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any new SQL strings have tests~

### Demo

<img width="1198" alt="screen shot 2019-03-01 at 1 58 46 pm" src="https://user-images.githubusercontent.com/16109558/53659728-3c54e380-3c2a-11e9-90e1-de59ae38c795.png">

## Testing Instructions

 * Go to v2 UI's project layer export list
 * Click on `New export` on top
 * Create an export on this export create UI with different export targets and resolutions
 * Use the AOI definition component
 * Make sure they create exports successfully
 * Run `rf export <exportID>` command to check that the exports created with this UI works

Closes https://github.com/raster-foundry/raster-foundry/issues/4607
